src/App.js

```diff
@@ -4,15 +4,18 @@ import Player from './components/Player';
 import CardList from './components/CardList';
 import { useState } from 'react';
 import ExecutePassButton from './components/ExecutePassButton';
+import { useContext } from 'react';
+import { UserContext } from './context/GameContext';
+
 
 function App() {
   const [deck, setDeck] = useState(initialCards);
   const [playerOneHand, setPlayerOneHand] = useState([]);
-  const [selectedCard, setSelectedCard] = useState();
   const [playerTwoHand, setPlayerTwoHand] = useState([]);
   const [playerThreeHand, setPlayerThreeHand] = useState([]);
-  const [from, setFrom] = useState('deck');
-  const [to, setTo] = useState(1);
+  const { to } = useContext(UserContext);
```

Comment:
You can destructure everything you're bringing in from context on the same line. For example, 
`const { to, from, setFrom, selectedCard, setSelectedCard } = useContext(UserContext);`